README.md

```diff
@@ -24,6 +24,9 @@ The Docker Compose file contains the following services:
 **Elasticsearch**  
 [Elasticsearch](https://www.elastic.co/products/elasticsearch) is a search engine based on [Lucene](https://lucene.apache.org/core/). It provides a distributed, multi-tenant capable full-text search engine with an HTTP web interface and schema-free JSON documents. Elasticsearch is developed in Java and is released as open source under the terms of the Apache License.
 
+**Audit Web Service**
+The Audit Web Service is a RESTful Web Service and provides a simple API. It can be used to index audit event messages into Elasticsearch. Each audit event message will comprise a set of fixed fields including the application the audit event message is associated with, the user that triggered the audit event as well as the tenant that the user belongs to. The audit event message will also include additional fields specific to the application. 
```

Comment:
I find this difficult to read:
"Each audit event message will comprise a set of fixed fields including the application the audit event message is associated with, the user that triggered the audit event as well as the tenant that the user belongs to."

Maybe something like:
"Each audit event message will comprise a set of fixed fields including the application, the user and the tenant which triggered the audit event."